src/core.c/CompUnit/Repository/Installation.rakumod

```diff
@@ -8,40 +8,24 @@ class CompUnit::Repository::Installation does CompUnit::Repository::Locally does
     has Int $!version;
     has $!precomp-store;    # cache for .precomp-store
     has $!precomp-stores;   # cache for !precomp-stores
+    has %!config;
 
     my $verbose = nqp::getenvhash<RAKUDO_LOG_PRECOMP>;
     my constant @script-postfixes = '', '-m', '-j', '-js';
     my constant @all-script-extensions =
         '', '-m', '-j', '-js', '.bat', '-m.bat', '-j.bat', '-js.bat';
 
-    my constant $windows-wrapper = Q/@rem = '--*-Perl-*--
-@echo off
-if "%OS%" == "Windows_NT" goto WinNT
-#raku# "%~dpn0" %1 %2 %3 %4 %5 %6 %7 %8 %9
-goto endofraku
-:WinNT
-#raku# "%~dpn0" %*
-if NOT "%COMSPEC%" == "%SystemRoot%\system32\cmd.exe" goto endofraku
-if %errorlevel% == 9009 echo You do not have Rakudo in your PATH.
-if errorlevel 1 goto script_failed_so_exit_with_non_zero_val 2>nul
-goto endofraku
-@rem ';
-__END__
-:endofraku
-/;
-
-    my constant $raku-wrapper = '#!/usr/bin/env #raku#
-sub MAIN(*@, *%) {
+    my constant $raku-wrapper-code = 'sub MAIN(*@, *%) {
     CompUnit::RepositoryRegistry.run-script("#name#");
 }';
 
-
-    method TWEAK() {
+    method TWEAK(:$wrapper-mode, :$wrapper-rakudo-dir) {
```

Comment:
Would be nice to have some type constraints on these params.

Reply:
Done.